bin/scpipe/lib/sc/pipe.rb

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env ruby
+#
+# SC:Pipe derived from sclang_pipe (SCVIM Package)
+# Copyright 2007 Alex Norman under GPL
+#
+# modified 2010 stephen lumenta
+# modified 2012 José Fernández Ramos
+
+require 'fileutils'
+require 'singleton'
+require 'tmpdir'
+
+module SC
+
+  @@sclang_path = `which sclang`
+
+  def self.sclang_path
+    if @@sclang_path.empty?
+      if File.exists?("/Applications/SuperCollider.app/Contents/Resources/sclang")
```

Comment:
@brianlheim Yeah that seems safer. Do you mind confirming the path for *< 3.7.2* and I'll change it right away?